style.css

```diff
@@ -161,25 +160,43 @@ i:hover {
 
 .work-card {
   display: flex;
+  position: relative;
   flex-direction: column;
-  border: 1px solid gray;
-  border-top: 300px solid #d2d2d2;
-  border-radius: 20px;
+  background-color: #d2d2d2;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
   margin: 1rem 1rem;
   text-align: center;
+  height: 474px;
+}
+
+.work-card:enabled {
+  border: 2px solid rgb(40, 184, 246);
+}
+
+.text {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+  border-radius: 8px;
+  background-color: white;
 }
 
 .work-card2 {
   display: flex;
   background-color: #ebf0ee;
   flex-direction: column;
-  border: 1px solid gray;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
```

Comment:
- [ ] You did a great job in styling your work-card section according to the design, But there are responsiveness issues I think you need to fix. 👍 💯 
- - On the screen sizes between `770px`  and `1110px` the content of the card tends to overflow, Please consider fixing this as it makes your portfolio not look professional at that point. I think you can fix this by either reducing the font sizes of the skills or you display only 2 cards at that screen size. 😄 
- Screenshot for reference:
![Screenshot 2023-03-31 002133](https://user-images.githubusercontent.com/107632780/228987677-1266f618-e657-4dc3-98e3-68b9137ae662.png)

Reply:
done @V-Blaze